Common/Utils/BossRushUtils.cs

```diff
@@ -98,7 +131,7 @@ public static partial class BossRushUtils {
 			}
 			return hostilePos;
 		}
-		public static bool LookForHostileNPC(this Vector2 position, out NPC npc, float distance, bool CanLockThroughTile = false) {
+		public static bool LookForHostileNPC(this Vector2 position, out NPC npc, float distance, bool CanLookThroughTile = false) {
```

Comment:
`canLookThroughTile` is preferred over `CanLookThroughTile`

Reply:
alright